src/content/blog/whats-new-october-2023.mdx

```diff
@@ -0,0 +1,124 @@
+
+[Code Fix: The SIBA Website's Move from Create-React-App to Astro](https://brittanisavery.com/post/move-siba-to-astro): read about converting a legacy CRA project to Astro by Brittani S. Avery
+
+[Translate routes in Astro with Content collection, subpage, optional trailingSlash and base support](https://www.webdesign-sopelnik.de/en/blog/translate-routes-for-astro-content-collections-or-subpages-with-trailingslash-and-base-support/): a guide to translating routes for content collections and subpages by Artur Sopelnik 
+
+## Showcase
+
+Each week on the Astro Community Call in Discord, we shine the spotlight on projects posted by community members in our `#showcase` channel from the following week. Here's a sample of what we saw this month!
+
+### Tools
+
+- [Darkmatter](https://getdarkmatter.dev/): A CMS-like interface for your Astro content collections (MacOS)
+- [Frontmatter CMS](https://frontmatter.codes/):  A Headless CMS right in your code editor - now supports Astro!
+- [Search Astro Documentation Raycast Extension](https://www.raycast.com/ElianCodes/search-astro-docs): search Astro docs directly within Raycast
+- [Set Up a Static Site With Astro on Kinsta](https://kinsta.com/docs/astro-static-site/): new guide for setting up a static site with Kinsta’s Application or Static Site hosting.
+
+### Themes
+
+- [Catonaut](https://github.com/AminoffZ/catonaut) by @aminoff
+- [Astro Supabase Starter](https://github.com/kevinzunigacuellar/astro-supabase) by @kevinzunigacuellar
+- [Engineering Blog](https://astro.build/themes/details/engineering-blog/) by @eddyvinck
+- [Ovidius](https://ovidius-astro-theme.netlify.app/) by @tomas_ba
+
+### Websites
+
+- [noobscience.rocks](https://www.noobscience.rocks/) by @noobscience
+- [fadhelmurphy.github.io](https://fadhelmurphy.github.io/) by @mamangpro
+- [rohitk06.vercel.app](https://rohitk06.vercel.app/) by @rohit06
+- [hvlxh.tk](https://www.hvlxh.tk/) by @hitesh_dev
+- [astronvim.com](https://astronvim.com/) by @uzaaft
+- [minov.studio](https://www.minov.studio/en/) by @suhaylmv
+- [linwood.dev](https://www.linwood.dev/) by @codedoctor
+- [shamann.dev](https://shamann.dev/) by @sylvhama
+- [paualbert.netlify.app](https://paualbert.netlify.app/) by @akond
+- [procastinatordev.vercel.app](https://procastinatordev.vercel.app/) by @procastinatordev
+- [mcdle.net](https://www.mcdle.net/) by @elitogame
+- [decimal.fm](https://www.decimal.fm/) by @chrsgrrtt
+- [dev.jeromeabel.net](https://dev.jeromeabel.net/) by @jeromeabel
+- [nunogois.com](https://www.nunogois.com/) by @nunogois
+
+### Starlight in the wild
+
+Need inspiration for your documentation site? Here are the new Starlight sites we spotted this month!
+
+- [docs.corsace.io](https://docs.corsace.io/en/)
```

Comment:
```suggestion
- [docs.launchfa.st/documentation](https://docs.launchfa.st/documentation)
- [docs.corsace.io](https://docs.corsace.io/en/)
```
Gotta throw Rishi some love! Plus they repost every time they're mentioned, so let's make it easy for them!